README.md

```diff
@@ -139,11 +155,12 @@ Releases of this project follow [semantic versioning](http://semver.org/), ie
 > - PATCH version when you make backwards-compatible bug fixes.
 
 To make a new version:
-- update `VERSION.txt` with the new version number
-- commit this change; the first line of the commit message **must** be in the
-  format `Bump version to X.X.X`
-- create a pull request for the version bump
+- run `yarn version` to update the version number;
+- (note that yarn has been configured **not** to create a new tag when you run this command - see `.yarnrc`)
+- if you are making a major change, also update the change log;
+- commit `package.json` and `CHANGELOG.md` if appropriate - for a small PR, this could be in the same commit as other
+  changes you are making; for a larger PR you might want a separate commit with a message that summarises the entire PR.
 
 When the pull request is merged
 [a Jenkins job](https://ci.beta.digitalmarketplace.service.gov.uk/view/Utils and toolkit/job/tag-toolkit/)
```

Comment:
We may as well fix this while we're here: the link is all broken. Not sure what the URI should be though (annoyingly I can't get on the VPN today so I can't look it up) - but the parentheses should just contain one URI, with no spaces in it :)

Reply:
(I bet someone at some point search-and-replaced `&` symbols with ` and ` !! :-o )

Reply:
I think it might be the link to that Jenkins job you sent me earlier: `https://github.com/alphagov/digitalmarketplace-jenkins/blob/master/job_definitions/toolkit.yml`

What do you think?

Reply:
I think it might be a link to that Jenkins job you sent me earlier, what do you think?

`https://github.com/alphagov/digitalmarketplace-jenkins/blob/master/job_definitions/toolkit.yml`